asv_bench/benchmarks/omnisci/benchmarks.py

```diff
@@ -256,25 +256,43 @@ def create_fillna_value(value_type: str, columns: list):
         return value
 
 
-class TimeValueCountsSeries:
-    param_names = ["shape", "ngroups"]
-    params = [
-        SERIES_DATA_SIZE[ASV_DATASET_SIZE],
-        GROUPBY_NGROUPS[ASV_DATASET_SIZE],
-    ]
-
-    def setup(self, shape, ngroups):
+class BaseTimeValueCounts:
+    def setup(self, shape, ngroups=5, subset=1):
```

Comment:
If the only change from modin on ray benchmarks is the data import triggering, could we just inherit from the modin-on-ray benchmarks without rewriting them all?

```python
from asv_bench.benchmarks.benchmarks import (
    BaseTimeValueCounts,
    TimeValueCountsSeries,
    TimeValueCountsFrame,
)

class BaseTimeValueCountsOmnisci(BaseTimeValueCounts):
    def setup(*args, **kwargs):
        super().setup(*args, **kwargs)
        trigger_import(self.df)

class TimeValueCountsSeriesOmnisci(BaseTimeValueCountsOmnisci, TimeValueCountsSeries):
    pass

class TimeValueCountsFrameOmnisci(BaseTimeValueCountsOmnisci, TimeValueCountsFrame):
    pass
```

Reply:
For series, we doesn't benchmark `bins` parameter.

In addition, benchmarks differ in data size.
For ray - `UNARY_OP_DATA_SIZE` also include `(5000, 5000)` shape.